tutorials/elixir-phoenix-on-google-compute-engine/index.md

```diff
@@ -179,7 +179,10 @@ Now you can create a release to test out your configuration.
 Building a release for deploying to the cloud is a bit more complicated
 because your build needs to take place in the same operating system and
 architecture as your deployment environment. In this section, you will use
-Docker to build a deployable release, and upload it to Google Cloud Storage.
+Docker to cross-compile a release that can run on Compute Engine VMs. (Docker
```

Comment:
To reduce # words, clarify tasks & reinforce tutorial goals, I suggest: 

In this section, you will use Docker only to cross-compile a release, which will run on a Compute Engine VM not a Docker container. Then, you will upload your built release to a Cloud Storage bucket.

Reply:
Shortened and re-focused the language.